### ISSUE TITLE
Number of epochs for training

### ISSUE THREAD

**Issue by JacobLiu-S** [pixelite1201/BEDLAM#28]
Dear BEDLAM team,

Thanks for your great work!
I have one question about the total training epochs. Since in configs/bedlam_hmr.yaml, the max_epochs is set to 200000 which is obviously not the real number of trained epochs. And I didn't find any early stoping callbacks (probably I miss it) in the repo. Could you provide how many epochs happened in training? And is the "200000" actually means iterations rather than "epochs"? 

Look forward to your reply and thanks in advance!

**Comment by pixelite1201** [comment]
Hello,
We use around 70-80 epochs for training. We use 3dpw-validation set and BEDLAM-validation set for early stopping when the loss saturated.

**Comment by MyGithubNotYours** [comment]
@JacobLiu-S If I understand your post correctly, I had the same question as you. I ended up looking at the pre-trained checkpoints that @pixelite1201 provided in this repository. For example, it looks like the pre-trained bedlam-cliff model was trained for 28 epochs.

```
import torch  

ckpt = torch.load("bedlam_cliff.ckpt")

ckpt.keys()
dict_keys(['epoch', 'global_step', 'pytorch-lightning_version', 'state_dict', 'callbacks', 'optimizer_states', 'lr_schedulers', 'hyper_parameters'])

ckpt['epoch']
28
```
